prime-dotnet-webapi/PrimeEnvironment.cs

```diff
@@ -109,5 +109,27 @@ public static class LdapApi
         {
             public static readonly string Url = GetEnvironmentVariable("LDAP_API_URL") ?? "https://common-logon-dev.hlth.gov.bc.ca/ldap";
         }
+
+        public static class PrimeSentryKeys
+        {
+            public static readonly string DevEnvDsn = "https://b10d3b20d6cc4eb39e0d90997c72fd0c@o988003.ingest.sentry.io/5945123";
+
+            // Value below can be changed for different environments
+            public static readonly double DevEnvTraceSampleRate = 1.0;
+
+            public static readonly string TestEnvDsn = "Placeholder for test DSN";
+
+            // The value below can be changed for different environments, arbitirarily set to 0.5 for test env.
+            // Meaning that each transaction will have a 50% chance of being sent to Sentry for performance
+            // monitoring in Test environment (higher percentage may cost performance)
+            public static readonly double TestEnvTraceSampleRate = 0.5;
+
+            public static readonly string ProdEnvDsn = "Placeholder for Prod DSN";
+            
+            // The value below can be changed for different environments, arbitirarily set to 0.2 for prod env.
+            // Meaning that each transaction will have a 20% chance of being sent to Sentry for performance
+            // monitoring in Test environment (higher percentage may cost performance)
+            public static readonly double ProdEnvTraceSampleRate = 0.2;
+        }
```

Comment:
Should these be environment variables?

Reply:
They could be but not necessarily.

Reply:
These should be environment variables rather than duplicating it this way. If these are not secrets, they can be in a deployment config.